File: app/src/main/java/com/meteoro/testeandroid/core/data/model/MoreInfo.java
package com.meteoro.testeandroid.core.data.model;

import com.google.gson.annotations.SerializedName;

import lombok.Getter;
import lombok.experimental.Accessors;

@Accessors(fluent = true, chain = true)
public class MoreInfo {

    @Getter
    FundInfo month;

    @Getter
    FundInfo year;

    @Getter
    @SerializedName("12months")
    FundInfo months12;
}

File: app/src/main/java/com/galdino/testandroid/data/entity/investment/MoreInfo.kt
package com.galdino.testandroid.data.entity.investment

import com.google.gson.annotations.SerializedName

data class MoreInfo (@SerializedName("month")
                val month: Month? = null,
                @SerializedName("year")
                val year: Year? = null,
                @SerializedName("12months")
                val months12: Months12? = null)